### PR TITLE
Bot telegram by  py new

### PR DESCRIPTION
**Pull request by Sherif-Ashraf421**

### CODE REVIEW

File: bot_telegram/botbutttton.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+list_btn = types.InlineKeyboardMarkup()
+list_btn2 = types.InlineKeyboardMarkup()
+list_btn3 = types.InlineKeyboardMarkup()
+b1= types.InlineKeyboardButton(text="yes,iam astra",callback_data="y")
+b2= types.InlineKeyboardButton(text="no",callback_data="n")
+b4= types.InlineKeyboardButton(text="time",callback_data="time")
+b5= types.InlineKeyboardButton(text="astra",callback_data="astra")
+b6= types.InlineKeyboardButton(text="EGYPT",callback_data="eg")
+b7= types.InlineKeyboardButton(text="UTC",callback_data="utc")
+list_btn.add(b1,b2)
```
Comment (MM150551) [inline bot_telegram/botbutttton.py:11]: test comment

**Review by MM150551 — COMMENTED** [review]
the code is decent

File: bot_telegram/botbutttton.py (hunk 1 above)
Comment (Sherif-Ashraf421) [reply bot_telegram/botbutttton.py:11]: back test comment

Comment (MM150551) [reply bot_telegram/botbutttton.py:11]: back back test comment

**Review by MM150551 — APPROVED** [review]

**Pull request merged**